lib/Cake/Test/Case/View/Helper/HtmlHelperTest.php

```diff
@@ -553,6 +553,26 @@ public function testImageTagWithTheme() {
 		}
 	}
 
+	/**
+	 * testBase64ImageTag method
+	 *
+	 * @return void
+	 */
+	public function testBase64ImageTag()
+	{
+		$this->Html->request->webroot = '';
+
+		//my apologies for the length of this...
+		$reallyLongBase64Result = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA1FJREFUeNqsVF1IU2EYfraF5U951P0wnTidP/kzm5GZuygNDSWhQRnECMohhgnRnRcpKgQSqGhCEIhLJhkUGhGu0JwiXqSYQyddhG6EG7bQ/E1ccfq+D3fYULvyufgO53vf85zn/RXxPI+jxDF6iEQi4aK0tNRAHtdUKpUuOjpaJ5fLoVAomG1hYQFbW1twuVy2tbW1GXL1YnBwcMb/LRUnYgchpESxsbFtZWVl6vz8fIHkMFDyoaEhjIyMUPKHlJhFSw9C1mCxWPiDMDExwbe0tLDn5ubmPju9a2pq4glHgaCwtraWb25uPlRNTU0NUbwEj+cvpNIMpKVpQaNISkpidpoGk8k009fXl8NySPP0P1RVVaGr6xGKr4SSt0UsLzvR2/sBP5bFkCs0jDwiIoITikJzUVxcDK1WeyAhvc/IuEoUfoRSKYFKFYbUFBnCwmQQIRLfvtnhdrtZccR739jM5ka0traSvy8fSGo0GjE1KcGobQcu5x9IJCGQiEOoMkhjmDi7oJDg1+3b8SQXIeju7sTurgSJiUmIi1NBrVaD4yKxsrKKrKw8zM+P4uvX3xgetiM9XYnTaYlYX/dRjplAQrvF4jFUVuaguroQoaEq5rS46ITD4YDX62VOHMeRFhPDYEiAWMQR+y7mHN9JyG4mKpDQubq6i8eP36O8nEdJSRxIUyMmRhoU9tjYGN6988Fm8yL33Cmcz03FpYtSPO3sR3vHc1sQoUYTgdzcbFitn/HmzSdSOT30ej1rjfDwcObk8/mQnHwSUVFhGHg7T1IzhzPZqVhye51Bo0fjHx//SRxXUF9/DxsbxzE15cDAwAArEp0KP/R6OfLy4lFYcBY7OycwPe0iud8RCIXRM5mu8/HxMYzMaLwFnS6P5SsQVqsVHR0dNq1Woc6/kKnOyswgefSirf1VIxm9BsrlV0j6aNtcUXH5zvZ2JNrbn5F26EVRURE0Gg0CFwTB6JMn5kI6+0ql/YFcFlXgr3CQQuLAJSTIFuvq7nMyWQomJ+dZlWdnZwWFZG5p+HeJGnPAdlKTd6d/2yBwHxKj7ubNG196erp4j8e9bxH09/ezJXDYiAaFTLG323I2NrYML1++Fnai337YFAVCdNQbW4wjxj8BBgDSNqdSpFi+eQAAAABJRU5ErkJggg==';
```

Comment:
[cake.icon.png is a default file in a native copy of the cake2 repo](https://github.com/cakephp/cakephp/blob/2.next/app/webroot/img/cake.icon.png). 

Granted, if the file is changed it will break the test, but then that's the nature of tests in general.